src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -124,6 +124,15 @@ interface WeakMap<K, V>{
     readonly [Symbol.toStringTag]: "WeakMap";
 }
 
+interface Set<T> {
+    [Symbol.iterator]():IterableIterator<T>;

```

Comment:
I think iterator is different, and should be in iterables and not in the wellknwon symbols.